pymc/sampling.py

```diff
@@ -181,14 +181,14 @@ def assign_step_methods(model, step=None, methods=STEP_METHODS, step_kwargs=None
     Parameters
     ----------
     model : Model object
-        A fully-specified model object
-    step : step function or vector of step functions
+        A fully-specified model object.
+    step : step function or vector of step functions, default=None
```

Comment:
```suggestion
    step : step function or iterable of step functions, optional
```

`None` is not used directly as default value, it is a placeholder to indicate the internal defaults should be used. Therefore, `optional` should be used instead as explained in https://pymc-data-umbrella.xyz/en/latest/sprint/docstring_tutorial.html#parameters. Not sure what vector means, but it seems like iterable can be used which is much more common and recommended by numpydoc.